src/views/DrawerNavigatorItems.tsx

```diff
@@ -54,13 +67,24 @@ const DrawerNavigatorItems = ({
       const scene = { route, index, focused, tintColor: color };
       const icon = renderIcon(scene);
       const label = getLabel(scene);
-      const accessibilityLabel = typeof label === 'string' ? label : undefined;
+      const accessibleLabel =
```

Comment:
I'd rather use something like `finalAccessibilityLabel` or something like that. `accessibleLabel` and `accessibilityLabel` can be easily confused in the first glance.